beamer/deploy/commands.py

```diff
@@ -0,0 +1,97 @@
+import subprocess
+from pathlib import Path
+from typing import Any
+
+import click
+import structlog
+from web3 import Web3
+from web3.gas_strategies.rpc import rpc_gas_price_strategy
+from web3.types import TxParams, Wei
+
+import beamer.contracts
+import beamer.deploy.artifacts
+import beamer.deploy.config
+import beamer.util
+from beamer.deploy.util import deploy_contract, get_commit_id
+from beamer.typing import ChainId
+from beamer.util import make_web3
+
+log = structlog.get_logger(__name__)
+
+
+def _margin_gas_price_strategy(web3: Web3, transaction_params: TxParams) -> Wei:
```

Comment:
nitpick, but I'm pretty sure that we can remove this strategy as it is not applied in rollups, and for our base chains we use the type 2 transactions. We can also remove the argument in `make_web3()`

Reply:
Good point. I remove this specific strategy, but left the `make_web3` function as-is because that arg is still used by the existing deployment script. We'll remove it once we remove the old deployment script.